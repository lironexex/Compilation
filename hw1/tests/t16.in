// illegal strings 7 - unclosed string 2
"this string ends now"and does not include this"
