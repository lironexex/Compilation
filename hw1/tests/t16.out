1 COMMENT 1
2 STRING this string ends now
2 ID and
2 ID does
2 ID not
2 ID include
2 ID this
Error unclosed string
